doc/book/examples/sources/basics/bag.move

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module 0x0::bag_example {
```

Comment:
Can we use a named address like `me` or `examples` here and elsewhere? It think this is a lot more readable/easier to understand--the `0x0` just raises so many questions.

Reply:
Yup!